eth/test/utils/TestWorld.ts

```diff
@@ -29,6 +29,29 @@ export interface InitializeWorldArgs {
   whitelistEnabled: boolean;
 }
 
+export async function createArena(
+  contract: DarkForest,
+  initializers: HardhatRuntimeEnvironment['settings']['darkforest']['initializers']
+): Promise<DarkForest> {
+  const { LibGameUtils } = await deployLibraries({}, hre);
+  const DFInitialize = await deployDiamondInit({}, { LibGameUtils }, hre);
+  const initFunctionCall = DFInitialize.interface.encodeFunctionData('init', [
+    false,
+    '',
+    initializers,
+  ]);
+
+  const newLobbyTx = await contract.createLobby(DFInitialize.address, initFunctionCall);
+  await newLobbyTx.wait();
+  const eventFilter = contract.filters.LobbyCreated();
+  const events = await contract.queryFilter(eventFilter, 'latest');
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { ownerAddress, lobbyAddress } = events[0].args;
```

Comment:
Why are you getting the ownerAddress here and not using it?

Reply:
Fixed.